src/main/java/org/sagebionetworks/web/client/widget/entity/WikiPageWidgetViewImpl.java

```diff
@@ -462,6 +462,8 @@ public void invoke() {
 				});
 				
 				add(markdownEditorWidget.asWidget());
+				//also add the wiki attachments dialog
+				add(wikiAttachments.asWidget());

```

Comment:
Get rid of wiki page modal, and instead incorporate into Insert Attachments and Insert Image wiki editors.